pkg/subctl/cmd/join.go

```diff
@@ -69,6 +61,51 @@ var joinCmd = &cobra.Command{
 
 func joinSubmarinerCluster(subctlData *datafile.SubctlData) {
 
+	// Missing information
+	var qs = []*survey.Question{}
+
+	if len(clusterID) == 0 {
```

Comment:
can we change it to...  len(clusterID)==0 or !isValidClusterID(str)  ?

In such way we avoid the final panic condition if the cmdline was wrong.

Reply:
Ah yes, I like that.